packages/adapter-dynamodb/src/index.ts

```diff
@@ -44,7 +44,7 @@ export interface DynamoDBAdapterOptions {
  * By default, the adapter expects a table with a partition key `pk` and a sort key `sk`, as well as a global secondary index named `GSI1` with `GSI1PK` as partition key and `GSI1SK` as sorting key. To automatically delete sessions and verification requests after they expire using [dynamodb TTL](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/TTL.html) you should [enable the TTL](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/time-to-live-ttl-how-to.html) with attribute name 'expires'. You can set whatever you want as the table name and the billing method.
  * You can find the full schema in the table structure section below.
  *
- * ## Configuring `pages/api/auth/[...nextauth].js`
+ * ## Configuring AuthJS
```

Comment:
I'm trying to make all the doc pages for the adapters look + or - the same 📐 

For the sections on how to configure the adapter in the AuthJS options, I think this heading is clearer, and we should say `AuthJS` rather than `NextAuth` right?

Reply:
*Auth.js